### DISCUSSION TITLE
Using Navigation with Bottom Sheet Modal

### DISCUSSION THREAD

**Discussion by james-two** [gorhom/react-native-bottom-sheet#1579] [Q&A]
Thank you for the amazing BottomSheet package 🚀

Is there a guide or example code for using 
Navigation with Bottom Sheet Modal?
https://gorhom.github.io/react-native-bottom-sheet/react-navigation-integration

Simple replacing Bottom Sheet for Bottom Sheet Modal isn't currently working for me.

Is a workaround such as https://github.com/gorhom/react-native-bottom-sheet/discussions/1178 necessary?

**Comment by jamezening** [comment]
(original poster here, replying from a different GH account)
After chewing on the example code, I think I understand how it works.

Is there a way to integrate the entire app's navigator without the entire app being showed in the BottomSheet.
If not, would such an app require 2 navigators - 1 for the app and 1 for within the BottomSheet?
